registry/storage/driver/base/base.go

```diff
@@ -89,8 +96,16 @@ func (base *Base) setDriverName(e error) error {
 
 // GetContent wraps GetContent of underlying storage driver.
 func (base *Base) GetContent(ctx context.Context, path string) ([]byte, error) {
-	ctx, done := dcontext.WithTrace(ctx)
-	defer done("%s.GetContent(%q)", base.Name(), path)
+	attrs := []attribute.KeyValue{
+		attribute.String("driver.name", base.Name()),
+		attribute.String("path", path),
```

Comment:
https://opentelemetry.io/docs/specs/semconv/general/attribute-naming/#recommendations-for-application-developers

For new names, perhaps `io.cncf.distribution` would be a good choice of prefix?

Reply:
Done.